packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/components/DrawingToolMarks/DrawingToolMarks.js

```diff
@@ -29,6 +29,7 @@ function DrawingToolMarks ({ activeMarkId, marks, onDelete, onDeselectMark, onFi
     }
 
     function moveMark (event, difference) {
+      if (!isActive) return;
```

Comment:
Looks good. Maybe this should use a callback prop, `onMoveMark`, like the other handlers? Then we can control draggable behaviour from the parent, and turn it off completely if necessary.